makefile

```diff
@@ -1,8 +1,11 @@
 VERSION := $(shell git describe --tags)
 
-linux:
-	GOOS=darwin GOARCH=386 go build -o ./dist/mgphoto-linux -ldflags="-X main.version=${VERSION}" ./*.go
+linux32:
```

Comment:
Thanks for this fix! I will make one suggestion though. Can you change the output names of these so they don't get overridden when running `make all`? I'd suggest `mgphoto-linux32` and `mgphoto-linux64` as output names. Also, when that's updated would you mind squashing into a single commit?

Thanks!

Reply:
Oh sure, missed that!! :+1: